prime-router/src/main/kotlin/transport/SftpTransport.kt

```diff
@@ -55,10 +56,10 @@ class SftpTransport : ITransport, Logging {
 
             // Dev note:  db table requires body_url to be unique, but not external_name
             val fileName = Report.formExternalFilename(header)
-            context.logger.info("Successfully connected to $sftpTransportType, ready to upload $fileName")
+            logger.info("Successfully connected to $sftpTransportType, ready to upload $fileName")
```

Comment:
Why do you delete context?  Please try not to change SftpTransport.kt.  You need to add only SftpTransportIntegrationTests.kt.  It should follow the same logic as RESTTransportIntegrationTests.kt

Reply:
There's 2 loggers being used in this class. 1 comes from `context` and the other comes from the `Logging` interface. I switched it over to the one from `Logging` so it would be readable in unit tests since the one from `context` is mocked out in the abstract `TransportIntegrationTests`.

Reply:
@jalbinson in that case, please use context.logger instead of logger.  We intended the context.logger to be mockable and it is for cloud context when it is live.  The logger was added by Christ Winters and he should use the context.logger.

Reply:
I reverted the change back to use `context.logger`.

Reply:
The SFTPTransportIntegrationTest is very good overall.  I want to add a few comments to them.  Therefore, whoever is looking at the test doesn't have to try to figure it out but just read your comments.  Great job and thank you.